### PR TITLE
docs: added recipes to consume SSE via POST req

### PR DESCRIPTION
**Pull request by jd-solanki**
> [!NOTE]
> I was unable to run the docs due to unknown issue and complexity of cloning two repos. So please run the docs with this PR and update if needed. Sorry for unable to test it in the docs 🙏🏻 . If possible can we've single repo for docs in v4 for easy contribution?

### 🔗 Linked issue

#26877

### 📚 Description

As discussed in above mentioned issue we'll mention recipe for consuming the SSE via post request I've added recipe for it in data fetching page.

### CODE REVIEW

**Comment by Lehoczky** [comment]
Yeah I would also support moving the docs into one repo, I started a discussion last year, though it didn't got any attention.

File: docs/1.getting-started/6.data-fetching.md [hunk 1]
```diff
@@ -623,3 +623,37 @@ const { data } = await useFetch('/api/superjson', {
...
+
+## Recipes
+
+### Consuming SSE (Server Sent Events) via POST request
+
+::tip
+If you're consuming SSE via GET request, you can use [`EventSource`](https://developer.mozilla.org/en-US/docs/Web/API/EventSource) or VueUse composable [`useEventSource`](https://vueuse.org/core/useEventSource/).
+::
+
+When consuming SSE via POST request, you've to handle the connection manually. Here's how you can do it:
```
Comment (danielroe) [inline docs/1.getting-started/6.data-fetching.md:635]:
```suggestion
When consuming SSE via POST request, you need to handle the connection manually. Here's how you can do it:
```

**Review by danielroe — APPROVED** [review]

**Comment by danielroe** [comment]
(No need to merge `main`, please. The reason I hadn't merged is to to reduce the number of times CI runs.)

**Comment by danielroe** [comment]
Thank you - nice addition to the docs! ❤️

**Comment by jd-solanki** [comment]
Thanks daniel, It's always nice to talk with you ❤️

**Pull request merged**